integration/container/run_linux_test.go

```diff
@@ -401,3 +401,48 @@ func TestSeccomp(t *testing.T) {
 		}
 	}
 }
+
+func TestCgroupRW(t *testing.T) {
```

Comment:
For the cases that don't error out currently, can we validate that cgroups rw/ro respectively?

Reply:
k, i've added that, but now am curious why the build of `dockerd` from this branch works for me on my machine with the command `docker run -it --rm --security-opt writable-cgroups=true debian mkdir /sys/fs/cgroup/foo`, but it fails in this test as permission denied read-only file system.

Reply:
hrm. I'm puzzling on why this works on the host, but not in the tests...